src/main/resources/db/migration/db.feedback.migration/V202409231227__create_feedback_and_identified_feedback_response.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE feedback (
+    feedback_id BIGSERIAL primary key,
+    company_id BIGINT NOT NULL,
+    department VARCHAR(30),
+    feedback VARCHAR(200) NOT NULL,
+    date_given DATE DEFAULT CURRENT_DATE,
```

Comment:
creation_date is a better name. Also, better use Timestamp rather than Date

Reply:
changed to creation_timestamp... fixed, thought timestamp isn't needed, but can see why.